Source/WebCore/platform/graphics/cg/UTIRegistry.cpp

```diff
@@ -78,6 +74,9 @@ const MemoryCompactLookupOnlyRobinHoodHashSet<String>& defaultSupportedImageType
             if (systemSupportedImageTypes.contains(imageType))
                 filtered.add(imageType);
         }
+        // rdar://104940377 Workaround for CGImageSourceCopyTypeIdentifiers not returning AVIF for iOS simulator
+        filtered.add("public.avif"_s);
+        filtered.add("public.avis"_s);
```

Comment:
This code will be enabled on down level macOS (i.e. Monetary) which is incorrect. I think it should be enabled only on iOS simulator 16.0.

Reply:
Fixed.